packages/desktop-client/src/components/mobile/transactions/FocusableAmountInput.tsx

```diff
@@ -216,7 +273,7 @@ export const FocusableAmountInput = memo(function FocusableAmountInput({
               borderBottomWidth: 1,
               borderColor: '#e0e0e0',
               justifyContent: 'center',
-              transform: [{ translateY: 0.5 }],
+              // transform: [{ translateY: 0.5 }],
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
```